app/services/invoice_custom_sections/deselect/for_organization_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module InvoiceCustomSections
+  module Deselect
```

Comment:
not sure if there's a need for a `Deselect` module 🤔 

I don't particularly mind either way, so just a nitpick.

Reply:
thank you, it actually is refactored in one of the further PRs 🙈